### DISCUSSION TITLE
VSIX Extensions install

### DISCUSSION THREAD

**Discussion by blarth** [stackblitz/docs#57] [Ideas]
Hello, 
I'm amazed by this tool, made my account like 15 minutes ago and it impressed me.
I'm missing a feature from Vscode, the VSIX extensions intall, do you guys think of having this feat or its not on the radar?

Have a great week!

**Comment by samdenty** [comment]
It on the radar along with adding support for custom registries (such as https://open-vsx.org/ or our own one). Currently you can only enable a predefined set of extensions

**Reply by blarth** [reply]
Thanks for the reply man!
